packages/manager/modules/dbaas-logs/src/logs/detail/osd/logs-osd.controller.js

```diff
@@ -0,0 +1,78 @@
+import template from './add/logs-osd-add.html';
+import datagridToIcebergFilter from '../logs-iceberg.utils';
+
+export default class LogsOsdCtrl {
+  /* @ngInject */
+  constructor(
+    $stateParams,
```

Comment:
**nitpick:** Avoid injecting `$stateParams` and use `resolve` method on the route definition to pass the params into the controller

Reply:
You are right, but it was already the case when this section was called "kibana". If it needs to be refactored, i suggest to do that in an other PR.